projectile.el

```diff
@@ -867,16 +867,31 @@ Files are returned as relative paths to the project root."
      ((eq vcs 'git) projectile-git-submodule-command)
      (t ""))))
 
-(defun projectile-get-all-sub-projects (project)
-  "Get all sub-projects for a given projects.
-PROJECT is base directory to start search recursively."
+(defun projectile-get-immediate-sub-projects (project)
+  "Get immediate sub-projects for a given project. Does not recurse.
+It is assumed that `project` will end with an appropriate path delimiter, such as '/' or a '\'"
```

Comment:
That seems excessive. (IE we wouldn't go and validate that a number was a number everywhere a number was expected).

It's just describing the expected schema.